sklearn/base.py

```diff
@@ -545,7 +545,7 @@ def __sklearn_tags__(self):
 
     def score(self, X, y, sample_weight=None):
         """
-        Return the mean accuracy on the given test data and labels.
+        Return mean :ref:`accuracy <accuracy_score>` on test data and labels.
```

Comment:
Thinking again about the use of the word 'mean' again, maybe it is used to indicate it is not the count (which you get with `normalize=False`) but the fraction/'mean' accuracy across the test set?

Following on from our conversation in https://github.com/scikit-learn/scikit-learn/pull/30316#discussion_r1858466559

I do still agree that 'mean' is confusing. Not sure what a better solution would be though, 'fraction accuracy' ?

Reply:
I've removed 'mean', but happy to change